integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -497,9 +480,9 @@ def test_hash_grpby_pivot(data_gen, conf):
 @approximate_float
 @ignore_order(local=True)
 @incompat
-@pytest.mark.parametrize('data_gen', _init_list_no_nans, ids=idfn)
+@pytest.mark.parametrize('data_gen', _init_list, ids=idfn)
 @pytest.mark.parametrize('conf', get_params(_confs, params_markers_for_confs), ids=idfn)
-def test_hash_grpby_pivot_without_nans(data_gen, conf):
+def test_hash_grpby_pivot_with_nans(data_gen, conf):
```

Comment:
This test appears to be a subset of test_hash_grpby_pivot. If so, it should be removed.